cases/drp-00-05.tex

```diff
@@ -3,16 +3,24 @@ \subsection{DRP-00-05: Execution of the DRP Science Payload by the DRP Execution
 
 \subsubsection{Requirements}
 
-DMS-REQ-0293,DMS-REQ-0294.
+DMS-REQ-0106,DMS-REQ-0292,DMS-REQ-0293,DMS-REQ-0294,DMS-REQ-0302,DMS-REQ-0303,DMS-REQ-0304.
 
 \subsubsection{Test items}
 
-This test will check:
+This test will check that the DRP Science Payload can be executed using the current state-of-the-art execution service provided by the LSST Data Facility.
 
-\begin{itemize}
+Specifically, this will check if the following requirements are fulfilled:
 
-  \item{That the DRP Science Payload can be executed using the current
-  state-of-the-art execution service provided by the LSST Data Facility.}
+\begin{itemize}
+  \item{For each Coadded Image, DMS shall store: the list of input images and the pipeline parameters, including software versions, used to derive it, and a sufficient set of metadata attributes for users to re-create them in whole or in part (DMS-REQ-0106).}
+  \item{To reduce the likelihood for confusion, all IDs shall be unique across databases and database versions, other than those corresponding to uniquely identifiable entities (i.e., IDs of exposures) (DMS-REQ-0292).}
```

Comment:
We thought of that too.  Many DMSR items are boarder than this test case is about. We listed those we find most relevant although part of the requirements ought to be fulfilled by some other parts of the system. 

I copied the original DMSR directly here. I'm tempted to delete them as they are duplicate as it is. Opinions?

Reply:
I agree that simply copying the DMSR here is of dubious usefulness.

My interpretation is that the "test items" should list the products/components under test, rather than the requirements which refer to them. So, for example, the test items might be "processing middleware" or "installation procedures" or a particular data product, rather than "DMS-REQ...".

This makes more sense in the context of a verification matrix: a requirement is satisfied by passing a number of test cases, each of which tests just some subset of the components & products which are necessary to meet that requirement.